api_docs/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 177**:
+
+* [`POST /register`](/api/register-queue):
```

Comment:
This should mention both of the endpoints that changed.

Reply:
Yes, I have added [`[POST /server_settings](/api/get-server-settings)`](https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/api_docs/changelog.md?plain=1#L26)